build/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM mcr.microsoft.com/playwright:v1.30.0
+
+RUN apt-get update &&\
+  apt-get install software-properties-common -y &&\
+  apt-add-repository universe -y &&\
+  apt-get update &&\
+  apt-get install openjdk-17-jdk -y
+
+ARG MAVEN_VERSION=3.8.7
+ARG USER_HOME_DIR="/root"
+ARG SHA=21c2be0a180a326353e8f6d12289f74bc7cd53080305f05358936f3a1b6dd4d91203f4cc799e81761cf5c53c5bbe9dcc13bdb27ec8f57ecf21b2f9ceec3c8d27
+ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
+
+RUN mkdir -p /usr/share/maven /usr/share/maven/ref \
+  && curl -fsSL -o /tmp/apache-maven.tar.gz ${BASE_URL}/apache-maven-${MAVEN_VERSION}-bin.tar.gz \
+  && echo "${SHA}  /tmp/apache-maven.tar.gz" | sha512sum -c - \
```

Comment:
looks hard to maintain (the SHA), do we really not trust these binaries and if they are corrupt, do we really care? :thinking:

Reply:
Copy/paste from the glsp client tests, but I also don't like it.
I add a sub task to the story and try to improve that later